test/docs.jl

```diff
@@ -21,206 +21,3 @@ end
 
 @test (@doc ModuleMacroDoc)    == "I am a module"
 @test (@doc ModuleMacroDoc.@m) == ["I am a macro"]
-
-# apropos function testing
```

Comment:
Sorry, missed that this is deleting not just the old copy of helpdb but also the old help system that was using it. Seems like a fair amount of the tests here should still be valid though?